src/schemas/json/install.json

```diff
@@ -389,7 +388,7 @@
             "title": "Name of the DNS Record.",
             "description": "For example sub for sub.example.com max length: 255"
           },
-          "contents": {
+          "content": {
```

Comment:
This changes the name of the property. Doesn't that immediately invalidate all existing `install.json` since the property name is changing? What's the purpose?

Reply:
This was originally a type it should have always been content not contents.